tests/playwright/src/model/pages/dashboard-page.ts

```diff
@@ -45,18 +57,26 @@ export class DashboardPage extends BasePage {
     this.heading = page.getByRole('heading', { name: 'Dashboard' });
 
     this.notificationsBox = this.content.getByRole('region', { name: 'Notifications Box' });
-    this.featuredExtensions = page.getByLabel('FeaturedExtensions');
+    this.featuredExtensions = page.getByRole('region', { name: 'FeaturedExtensions' });
 
-    this.devSandboxProvider = page.getByLabel('Developer Sandbox Provider');
+    // Dev Sandbox locators
+    this.devSandboxProvider = page.getByRole('region', { name: 'Developer Sandbox Provider' });
     this.devSandboxBox = this.featuredExtensions.getByLabel('Developer Sandbox');
-    this.devSandboxEnabledStatus = this.devSandboxProvider.getByText('RUNNING');
+    this.devSandboxActualState = this.devSandboxProvider.getByLabel(this.ACTUAL_STATE);
+    this.devSandboxStatusLabel = this.devSandboxActualState.getByLabel(this.CONNECTION_STATUS_LABEL);
 
-    this.openshiftLocalProvider = page.getByLabel('OpenShift Local Provider');
+    // OpenShift Local locators
+    this.openshiftLocalProvider = page.getByRole('region', { name: 'OpenShift Local Provider' });
     this.openshiftLocalBox = this.featuredExtensions.getByLabel('OpenShift Local');
-    this.openshiftLocalEnabledStatus = this.openshiftLocalProvider.getByText('NOT-INSTALLED');
-    this.initilizeAndStartButton = page.getByRole('button', { name: 'Initialize and start ' });
-    this.transitioningState = page.getByLabel('Transitioning State');
-    this.podmanMachineConnectionStatus = page.getByLabel('Connection Status Label');
+    this.openshiftLocalActualState = this.openshiftLocalProvider.getByLabel(this.ACTUAL_STATE);
+    this.openshiftLocalStatusLabel = this.openshiftLocalProvider.getByLabel(this.CONNECTION_STATUS_LABEL);
+
+    // Podman/Machine Provider locators
+    this.podmanProvider = page.getByRole('region', { name: 'Podman Provider' });
+    this.podmanInitilizeAndStartButton = this.podmanProvider.getByRole('button', { name: 'Initialize and start ' });
+    this.transitioningState = this.podmanProvider.getByLabel('Transitioning State');
+    this.podmanActualState = this.podmanProvider.getByLabel(this.ACTUAL_STATE);
```

Comment:
I'm not sure I understand whatt he goal of this field is, why do we need it? It seems to me the connection status label from below would work just as well from podmanProvider field, I'm not sure I understand the need to abstract so many middle layers.

Reply:
There could be more providers with the same button available in the Dashboard (openshift local). So it is better to hierarchically group them. You have provider region (big tiles on dashboard) for podman, dev sandbox, etc., then provider have these elements like buttons and statuses. It is much more readable and logically tied together.

Reply:
But there can't be more fields with the "podman Provider" label, as such any field that is extracted from podmanProvider would be unique, as such abstracting the Actual State and then the connection label is just adding more abstractions for no benefit.

My point is that "Actual state" field has no function and that extracting connection label from podmanProvider is sufficient to never be a duplicate.

Reply:
I would also argue against the idea that it's more readable, to me it seems like we're defining way to many fields that serve no practical purpose not only duplicating a lot of code but actively making it more confusing to understand to somebody that is not familiar with it.

For example how is somebody on first look supposed to know the difference between "Actual state" and "Status label" fields? There's no way, because they are practically identical from an user perspective.

Reply:
It will not work if you install any other extension adding a "Provider" box into a dashboard, since there will be two same elements defined by one locator. So we need to level down.

Reply:
How can it not? the parent element is "Podman Provider" adding another podman provider is not possible, there is only one box for podman as such it's always going to be unique. I really don't see how your hypothetical is possible.

Reply:
gosh, I though you are pointing at something else. Sorry, now I got it. You are actually pointing at the next line having one necessary layer. Sure thing, let's remove it.